sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java

```diff
@@ -1873,6 +1883,51 @@ interface WithAdditionalCapacities {
             WithCreate enableHibernation();
         }
 
+        /** The stage of the definition allowing to specify the SecurityType for the virtual machine. */
+        interface WithSecurityTypes {
+            /**
+             * Enables trusted launch.
+             * <p>
+             * Trusted launch only supports generation 2 VMs.
+             * </p>
+             *
+             * @see <a href="https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#limitations">Limitations</a>
+             *      for supported VM sizes, images and regions.
+             * @return the next stage of the definition
+             */
+            WithSecurityFeatures withTrustedLaunch();
+        }
+
+        /** The stage of the definition allowing to specify the security features for the virtual machine. */
+        interface WithSecurityFeatures extends WithCreate {
+            /**
+             * Enables secure boot feature.
+             *
+             * @return the next stage of the definition
+             */
+            WithSecurityFeatures withSecureBoot();
+
+            /**
+             * Disables secure boot should be disabled on the virtual machine.
+             *
+             * @return the next stage of the definition
+             */
+            WithSecurityFeatures withoutSecureBoot();
```

Comment:
Do we need "without" in create flow?

Reply:
Actually... No... If no specified, they are false by default. Removed, thanks.